docs/reference/search/request/search-type.asciidoc

```diff
@@ -50,6 +50,11 @@ During the second phase, the coordinating node requests the document
 content (and highlighted snippets, if any) from *only the relevant
 shards*.
 
+[source,js]
+--------------------------------------------------
+GET twitter/_search?search_type=query_then_fetch
+--------------------------------------------------
+
```

Comment:
Let's make this example executable/testable by adding `CONSOLE`. This ensures that the snippet will be executed/tested. We will also need to setup the index with data for the test and for this we can use `TEST[setup:twitter]`.

So, the overall snippet can look like this:
```
[source,js]
--------------------------------------------------
GET twitter/_search?search_type=query_then_fetch
--------------------------------------------------
// CONSOLE
// TEST[setup:twitter]
```